showcase/plot/clustered-stacked-bar-chart.js

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
Fixed - probably need a separate PR for a bulk find+replace for this, there's a bunch of 2016 references in the repo.